certified-connectors/Document Merge/apiDefinition.swagger.json

```diff
@@ -0,0 +1,191 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "DocumentMergeWebAPI",
+    "description": "SharePoint Document and Mail Merge is the simple, fast and clean way to merge SharePoint list data into predefined Word, Excel and PowerPoint document templates with NO CODE!",
+	"contact": {
```

Comment:
@cirrussoft please make sure that the indentation is correct.

Reply:
Fixed the issue